src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -941,192 +1024,205 @@ void func_80B161C0(EnTa* this, PlayState* play) {
         }
     }
 }
 
-void func_80B16504(EnTa* this, PlayState* play) {
-    u16 faceReaction = Text_GetFaceReaction(play, 0x18);
+void EnTa_IdleSittingInLonLonHouse(EnTa* this, PlayState* play) {
+    u16 faceReaction = Text_GetFaceReaction(play, TALON_FACE_REACTION_SET);
 
-    func_80B13AAC(this, play);
+    EnTa_SetTextForTalkInLonLonHouse(this, play);
 
-    if (func_80B142F4(this, play, this->actor.textId)) {
+    if (EnTa_TalkToPlayer(this, play, this->actor.textId)) {
         Audio_PlayActorSfx2(&this->actor, NA_SE_VO_TA_SURPRISE);
 
         if (faceReaction != 0) {
-            func_80B14FAC(this, func_80B15E28);
+            EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_TalkGeneralInLonLonHouse);
         } else {
-            SET_INFTABLE(INFTABLE_7E);
+            SET_INFTABLE(INFTABLE_TALKED_TO_TALON_IN_RANCH_HOUSE);
 
             switch (this->actor.textId) {
-                case 0x207E:
-                case 0x207F:
-                    func_80B14FAC(this, func_80B161C0);
+                case 0x207E: // First time talking in Lon Lon House
+                case 0x207F: // Play cucco game
+                    EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_WaitForPlayCuccoGameResponse);
                     break;
-                case 0x208B:
-                    func_80B14FAC(this, func_80B15FE8);
+                case 0x208B: // Play cucco game or buy milk
+                    EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_WaitBuyMilkOrPlayCuccoGameResponse);
                     break;
                 default:
-                    func_80B14FAC(this, func_80B16364);
+                    EnTa_SetActionFuncWithWakeUpAnimation(this, EnTa_ContinueTalkInLonLonHouse);
                     break;
             }
         }
     }
-    this->unk_2E0 &= ~1;
+    this->stateFlags &= ~TALON_STATE_FLAG_TRACKING_PLAYER;
 }
 
-void func_80B16608(EnTa* this, PlayState* play) {
+void EnTa_IdleAfterCuccoGameFinished(EnTa* this, PlayState* play) {
     if (Actor_ProcessTalkRequest(&this->actor, play)) {
         switch (this->actor.textId) {
-            case 0x2085:
-                this->actionFunc = func_80B161C0;
+            case 0x2085: // Retry?
+                this->actionFunc = EnTa_WaitForPlayCuccoGameResponse;
                 break;
-            case 0x2086:
-                this->actionFunc = func_80B162E8;
+            case 0x2086: // Initial win
+                this->actionFunc = EnTa_WaitForMarryMalonResponse;
                 break;
-            case 0x2088:
-                this->actionFunc = func_80B1642C;
+            case 0x2088: // Later win
+                this->actionFunc = EnTa_TalkAfterCuccoGameWon;
                 break;
         }
+        // Disable immediate automatic talk
         this->actor.flags &= ~ACTOR_FLAG_16;
     } else {
+        // Enable immediate automatic talk
         this->actor.flags |= ACTOR_FLAG_16;
```

Comment:
same reasoning as my previous comment, I think this can just left uncommented on for the short term (hopefully we get actor flag docs in this year copium)

Reply:
copium